core/eth/contracts.go

```diff
@@ -1,23 +1,34 @@
 package eth
 
 import (
-	"chainlink/core/logger"
 	"fmt"
+	"math/big"
+	"reflect"
 	"strings"
 
+	"chainlink/core/logger"
+
 	"github.com/ethereum/go-ethereum/accounts/abi"
 	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
 	"github.com/gobuffalo/packr"
 	"github.com/pkg/errors"
 	"github.com/tidwall/gjson"
 )
 
+type Contract interface {
```

Comment:
**Review notes:** converted this to an interface so that it could be embedded in other interfaces.